app/controllers/pages_controller.rb

```diff
@@ -1,6 +1,15 @@
 class PagesController < ApplicationController
-  skip_before_action :authenticate_user!, only: [:home]
+  skip_before_action :authenticate_user!, only: [:home, :index, :show, :book]
```

Comment:
@DelphineHelie 
delete index, show, book methods and related skip_before_action